kolibri/plugins/coach/assets/src/views/coach-exercise-render-page/attempt-log-list.vue

```diff
@@ -1,44 +1,41 @@
 <template>
 
-  <div class="answer-history">
+  <div class="attempt-log-list">
     <h3 class="header">{{ $tr('header') }}</h3>
 
     <ul class="history-list">
       <template v-for="(attemptLog, index) in attemptLogs">
-        <li v-if="index === 0">
-          <p class="item">
-            {{ daysElapsedText(attemptLog.daysElapsed) }}
-          </p>
-        </li>
+        <!--p v-if="index === 0" class="item">
+          TODO modify elapsed-time to do Days only?
```

Comment:
@rtibbles thoughts on this? [Here's the component in question](kolibri/plugins/coach/assets/src/views/elapsed-time/index.vue) I was thinking about having a `lowestTimeMeasurement` prop or something, so that we can set it to "Day" or any other measure. Default would be set to seconds so that nothing breaks